### ISSUE TITLE
Revise node limit inconsistencies

### ISSUE THREAD

**Issue by confusedbudgie** [ErinaSugino/Starbound-SBR-Mod#17]
Every time I try to start a threesome with the Tier 2 lounge item, The actors become invisible. I would use my Aphrodite bow to trigger the arousal of the two other participants. They would go to the bed with the lounge item and perform as normal. My player character can join, since its a Tier 2 lounge item. However when I join, it makes the two NPCs and my own character invisible. Even the position UI doesn't even appear. Even when I get to the bed first, only one of the NPCs would interact, as if it was just a Tier 1 addon. All I can do is move away from the threesome sex scene. Even when I do that, the two other actors remain invisible.

[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/11182125/starbound.log)

However, The Tier 1 lounge item works perfectly fine. The two actors can perform, and they can be seen. When my character is involved with another, the UI works as well.

EDIT: I have realized that Back in loverslab, I made a mistake or two with the tier lounges. I know I said that tier 2 was fine, but I meant tier 1, and tier 3 was supposed to be tier 2. Tier 3's lounge addon didn't have animations for the bed yet. I apologize if I caused any confusion, Creator! 

In conclusion:
Tier 1 is fine (Max actors: 2)
Tier 2 is causing issues (Max actors: 3)
Tier 3 seemed to not have animations yet (Max actors: 4)

**Comment by BeebBeebBoob** [comment]
As I see by myself I think three actors isn't at all implemented yet. 
Yes, there's code for three actors to join and yes, NPCs do join. 
But that's the most that was made. 
There's no artwork, animations and coding for manipulation.
That's why when we do get three actors (As I think), UI tries to get working animations from the code( their name, ID and clickable button so you could change). He tries but then just stop working cause the bugs.

We could only hope that someone will actually make, coding, artwork(animation) and testing.
For now, only two actors. (And 1 actor idle)

**Comment by ErinaSugino** [comment]
What furniture are you using the Addon on?

**Comment by ErinaSugino** [comment]
- [ ] Fix inconsistencies of addons always using 2 actor soft-limitations regardless of Tier
- [ ] Rework way available loungable nodes are managed to make soft-limitation hard cap
- [ ] Ensure hard cap is maintained (currently is just cancles actor creation loop, most like resulting in the bug as the actor for the player is never initialized and hence the UI never opened)

**Comment by confusedbudgie** [comment]
> What furniture are you using the Addon on?

I do apologize for my disappearance for so long! The furniture I was using for was a bed for a threesome with the tier 2 lounge item, which was stated that was the lounge item to be used for threesomes or less.
